includes/classes/order.php

```diff
@@ -1264,11 +1264,10 @@ function create_add_products($zf_insert_id = null, $zf_mode = false)
                 (!empty($this->products_ordered_attributes) ? "\n" . '<nobr>' . '<small><em>' . nl2br($this->products_ordered_attributes) . '</em></small>' . '</nobr>' : '') .
                 '</td>' . "\n" .
                 '<td class="product-details-num" valign="top" align="right">' .
-                $currencies->display_price($this->products[$i]['final_price'], $this->products[$i]['tax'], $this->products[$i]['qty']) .
+                $currencies->display_price($this->products[$i]['final_price'], $this->products[$i]['tax'], $this->products[$i]['qty']) . '</td>' . "\n" . '</tr>' . "\n" .
```

Comment:
Please skinny this PR down so that this is the only change.

Reply:
ok, didn't notice the whitespace